lib/types/application/ApplicationBuilder.js

```diff
@@ -16,7 +17,13 @@ const tasks = { // can't require index.js due to circular dependency
 };
 
 class ApplicationBuilder extends AbstractBuilder {
-	addStandardTasks({resourceCollections, project}) {
+	addStandardTasks({resourceCollections, project, log}) {
+		if (!project.metadata.namespace) {
+			log.info("Skipping some tasks due to missing application namespace information. If your project contains" +
+					"a Component.js, you might be missing a manifest.json file. " +
```

Comment:
Missing space in the log text "contains**a** Component.js"

Reply:
Done